src/Components/SingleArticle.jsx

```diff
@@ -0,0 +1,54 @@
+import { useParams } from "react-router"
+import { fetchArticleById } from "./Api"
+import { useState, useEffect } from "react"
+import Comments from './Comments'
+
+const SingleArticle = () => {
+
+    const { article_id } = useParams()
+
+    const [articleById, setArticleById] = useState('')
+    const [isloading, setIsLoading] = useState(false);
+    const [error, setError] = useState(false)
+
+    useEffect(() => {
+        setIsLoading(true)
+        fetchArticleById(article_id)
+          .then((fetchedArticle) => {
+            setArticleById(fetchedArticle)
+            setIsLoading(false)
+            if (shows.length === 0) {
+                setError(true);
+              }
+          })
+          .catch((err) => {
+            setIsLoading(false);
+            setError(true);
+          });
+      }, [])
+
+      if (isloading) {
+        return <p>Loading...</p>
+      }
+
+    return(
+        <>
+        <section>
+        <div>
+          <ul>
+          <h2>{articleById.title}</h2>
+          <p>Topic: {articleById.topic}</p>
+          <img className="image_single_article" src={articleById.article_img_url} alt='picture missing!' />
+          <p>{articleById.body}</p>
+          <p>Votes: {articleById.votes}</p>
+          <p>Written by {articleById.author}</p>
+          </ul>
+        </div>
+        {error && <p>Sorry, article not found !</p>}
+      </section>
+      <Comments commentCount={articleById.comment_count}/>
+      </>
```

Comment:
I would prefer to see a `Comments` component being created and implemented on the ticket for viewing comments, it isn't apart of this ticket